src/platform/packages/shared/content-management/table_list_view_table/src/components/table_sort_select.tsx

```diff
@@ -51,13 +51,13 @@ const i18nText = {
   updatedAtAscSort: i18n.translate(
     'contentManagement.tableList.listing.tableSortSelect.updatedAtAscLabel',
     {
-      defaultMessage: 'Least recently updated',
+      defaultMessage: 'Old to New',
```

Comment:
I saw the original ask and the wording: 

> Can we standardize this closer to the default datagrid sort, "Old-New, New-Old" or something like this.

And I thought this needs more discussion because "Old to New" is not technically very correct label. 

@timductive, Wanted to confirm if we should proceed with this change